.travis.yml

```diff
@@ -2,8 +2,6 @@ language: node_js
 before_install:
   - npm install -g npm@latest
 node_js:
-  - "0.12"
-  - "0.11"
-  - "0.10"
+  - "6.5.0"
```

Comment:
Is there a reason this is locked down to 6.5.0 (current LTS is 6.10.3)? Can we select for the major versions, plus current stable here (4, 6, stable), to ensure this works across all widely used versions?